website/docs/components/app-footer/partials/code/how-to-use.md

```diff
@@ -51,7 +52,44 @@ You can add custom links in addition to or instead of including `LegalLinks`.
 </Hds::AppFooter>
 ```
 
+### ExtraBefore & ExtraAfter
+
+Custom content can be added either before or after the `AppFooter` main content. The layout of this content is inline with other content by default.
+
+```handlebars
+<Hds::AppFooter as |AF|>
+  <AF.ExtraBefore>
+    <Doc::Placeholder @text="Extra Content Before" @height="2em" {{style width="fit-content" color="#636363" background="#ffd6d6"}} />
```

Comment:
@KristinLBradley I _think_ this is the first time we use inline styles in the code snippets provided to consumers. Usually we hide the styles under a custom class (with some form of "semantic" naming to suggest what it does). The reasons for this are:
- some consumers may not have the `{{style}}` modifier (it's an Ember addon) installed as dependency, and if they copy&paste the code it will not work, and they may not understand why it is so
- we don't want to suggest to use inline styles in their applications in this way (they may have other conventions to use/follow in their teams)

(for context: we use profusely `{{style}}` in the showcase, because is an internal-facing codebase, meant only for us (as code, I mean)

Reply:
Thanks. Sorry about that. I will open another PR.